app/views/devise/two_factor_authentication/show.html.erb

```diff
@@ -0,0 +1,26 @@
+<% content_for :title, "Check your phone" %>
+
+<%= form_with(url: "/admin/two-factor-authentication", html: { method: :put }) do |f| %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      <h1 class="govuk-heading-l">
+        <%= content_for(:title) %>
+      </h1>
+
+      <p class="govuk-body">We’ve sent you a text message with a security code.</p>
+
+      <%= f.govuk_number_field :code,
+          label: { text: "Security code" },
+          width: 5,
+          autofocus: true
```

Comment:
```suggestion
          width: 5,
          autocomplete: 'one-time-code',
          autofocus: true
```

Reply:
Why would you want that?

Reply:
* https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete
* https://dev.to/maciejtrzcinski/html-autocomplete-one-time-code-357g

Where supported, makes inputting number a bit quicker.

Reply:
In general sure but I'm not sure that should apply to an SMS OTP code (what help could/should the browser provide there?).

I'd say that falls more into the category of:

> Set autocomplete="off" for sensitive form fields, like credit card number
> Why: So other users of the browser can’t access this saved information

(https://github.com/ankane/secure_rails)

Reply:
I guess unlike credit cards, this value has a limited lifespan.